resources.nix

```diff
@@ -367,6 +367,15 @@
         doCheck = false;
       };
 
+      hermes_1_5_1 = pkgs.rustPlatform.buildRustPackage {
```

Comment:
Let's just update the `hermes` package to 1.5.1, and remove this. No one should ever need to run 1.0.0 anymore.

Reply:
it will require update toml builder which is not comptible with 1.0.  i do  not use builder, and in future will have builder somewhat tuned to my scenario i guess.